.github/CODEOWNERS

```
@@ -25,6 +25,7 @@
 /packages/jellyfish-transaction/src/script/defi/            @fuxingloh @canonbrother @ivan-zynesis @monstrobishi
 
 /packages/jellyfish-transaction-builder/        @fuxingloh @ivan-zynesis @monstrobishi
+/packages/jellyfish-transaction-signature/      @ivan-zynesis
```

```suggestion
/packages/jellyfish-transaction-signature/      @fuxingloh @ivan-zynesis
```

too dangerous to go solo for this

---

Oh I forgot it affect auto merge.